ols/app/models/config.py

```diff
@@ -74,6 +74,7 @@ class ProviderConfig(BaseModel):
     name: Optional[str] = None
     url: Optional[str] = None
     credentials: Optional[str] = None
+    project_id: Optional[str] = None
```

Comment:
What about having an `extra` section within the provider for these provider-specific configurations?

Reply:
Ok, I see the discussion above now (I didn't see the comments originally).